src/test/java/seedu/address/logic/parser/BookShelfParserTest.java

```diff
@@ -49,6 +50,30 @@ public void constructor_null_throwsNullPointerException() {
         new BookShelfParser(null);
     }
 
+    //@@author fishTT
+    @Test
+    public void caseInsensitive_success() throws Exception {
+        /* Case: firstCharUppercase_success() -> added */
```

Comment:
This and the following 3 comments don't make sense in this context.
Change to something like `Case: first character is uppercase`